ui/src/router/index.js

```diff
@@ -215,6 +215,13 @@ router.beforeEach((to, from, next) => {
       Cookies.remove("user");
 
       return next({ name: "general-landing" });
+    } else if (to.name == "general-user-delete-thankyou") {
```

Comment:
Rather than putting the handler here, its better to access the store from the Vue component instead. You can access the store by using this.$store; it will give the same output with importing the store from @store/index.js; 

You can also import the Cookies from "js-cookie", alternatively (better), register cookie as a new plugin in plugins/index.js and set it as Vue.prototype; that way we can use the Cookie functionality across the components.

```
import Vue from "vue";
import Cookies from "js-cookie";

Vue.prototype.$cookies = Cookies;

```

and in plugins/index.js add this line: 
```

import "./js-cookie";

```

Reply:
I have moved the handler from router to the user-settings page. In the user-settings page, I have added:
```
import Vue from "vue";
import Cookies from "js-cookie";
Vue.prototype.$cookies = Cookies;
```

And in onDeleteConfirmation() I have added:
`Vue.prototype.$cookies.remove("user");`

So far, everything seems to be working fine. Deleting the user account caused the user to get 'logged out' into the user-delete-thanks page. However, I did not import js-cookie into plugins/index.js. Adding that line caused the following error:
<img width="672" alt="err" src="https://user-images.githubusercontent.com/64476430/173020190-ce7a41e1-de12-44df-8727-be83d984ea28.png">

Reply:
Yes, that error was caused by the file being nonexistent. Sorry for the incomplete instruction;  when I said you should register  new plugin, you should make a new file for that plugin and paste the lines to register the $prototype.cookies into the file, then import it to index.js. Else it will return error of course; after all the file is nonexistent. :D

Reply:
So there is one file you need to create in plugins/ folder, a js file named js-cookie. 
Copy the line of code to register the plugin there, and then import the file in plugins/index.js

Reply:
Okay, so plugins/js-cookie.js looks like this
```
import Vue from "vue";
import Cookies from "js-cookie";

Vue.prototype.$cookies = Cookies;

```
and plugins/index.js gets a new line `import "./js-cookie";`
and in user settings view i can do it like this: `this.$cookies.remove("user");`

is that correct?

Reply:
Yeah, exactly like that 😀